### ISSUE TITLE
Java 13 not supported

### ISSUE THREAD

**Issue by vadim0x60** [metosin/kekkonen#51]
Kekkonen depends on outdated `[ring-middleware-format "0.7.2"]` which depends on outdated `[org.flatland/ordered "1.5.3"]` which is incompatible with modern versions of java, leading to the following exception on java 13

```
java.lang.IllegalArgumentException: Must hint overloaded method: toArray
        at clojure.lang.Compiler$NewInstanceMethod.parse(Compiler.java:8496)
        at clojure.lang.Compiler$NewInstanceExpr.build(Compiler.java:8058)
        at clojure.lang.Compiler$NewInstanceExpr$DeftypeParser.parse(Compiler.java:7934)
        at clojure.lang.Compiler.analyzeSeq(Compiler.java:7106)
```

**Comment by vadim0x60** [comment]
Addressed in [pull request](https://github.com/metosin/kekkonen/pull/52)

**Comment by miikka** [comment]
Okay, it should work now. Thanks for the PR. Released as `[metosin/kekkonen "0.5.0"]`.
